src/setupTests.js

```diff
@@ -41,3 +41,17 @@ Object.defineProperty(global.self, 'crypto', {
     getRandomValues: arr => crypto.randomBytes(arr.length),
   },
 })
+
+/**
+ * Jest is very leaky...
+ * see
+ * - https://github.com/facebook/jest/issues/7311
+ * - https://github.com/facebook/jest/issues/7874
+ *
+ * Note that global.gc will only be exposed if node is passed --expose-gc
+ */
+afterEach(() => {
+  if (global.gc) {
+    global.gc()
```

Comment:
Pleased that you found a way that does not clutter the codebase with these forced garbage collection calls!